README.md

```diff
@@ -555,6 +555,28 @@ app.playbackController('NextCommandIssued', (request, response) => {
 
 Note that some device interactions don't always produce PlaybackController events. See the [PlaybackController Interface Introduction](https://developer.amazon.com/public/solutions/alexa/alexa-skills-kit/docs/custom-playbackcontroller-interface-reference#introduction) for more details.
 
+### CanFulfillIntent
+
+Define the handler for canFulfillIntent used for skill discovery.  For instance the [Implementation instructions](https://developer.amazon.com/docs/custom-skills/implement-canfulfillintentrequest-for-name-free-interaction.html).
+
+See detailed explanation on the Intent and Slot [logic](https://developer.amazon.com/docs/custom-skills/understand-name-free-interaction-for-custom-skills.html).
+
+
+```javascript
+app.displayElementSelected(function(request, response) {
+  // The request object will return CanFulfillIntent object 
+  let canFulfillIntent =  request.getCanFulfillIntent();
```

Comment:
I'm not exactly sure what this code is doing — why are you grabbing this variable and then never accessing it?

Reply:
I also wonder if it'd be useful to have an example that uses a normal intent instead of a display intent(?). As someone who's only worked on voice skills, I'm having a hard time wrapping my head around how this example relates to existing intent handler code I've written.

Reply:
Thanks, the new changes make it much clearer how this works as currently implemented :)